rotkehlchen/data_migrations/migrations/migration_6.py

```diff
@@ -0,0 +1,80 @@
+import json
+import logging
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import SupportedBlockchain
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+DEAD_NODES_RPCS = [
+    'https://api.mycryptoapi.com/eth',  # mycrypto
+    'https://mainnet-nethermind.blockscout.com/',  # blockscout
+    'https://mainnet.eth.cloud.ava.do/',  # avado
+    'https://web3.1inch.exchange',  # 1inch
+    'https://names.mewapi.io/rpc/eth',  # myetherwallet
+]
+
+
+def update_nodes_in_database(write_cursor: 'DBCursor', rotki: 'Rotkehlchen') -> None:
+    dir_path = Path(__file__).resolve().parent.parent.parent
```

Comment:
Can you add a docstring explaining what this migration does and the two possible cases